content/en/how-to-guides/internal-guides/change_management.adoc

```diff
@@ -24,11 +24,11 @@ These guidelines assume that you're familiar with the GitHub workflow.
 
 == Creating an issue ==
 
-If you've found a problem in the docs, please create an issue in the https://github.com/tetrabiodistributed/project-tetra-docs/issues[Project Tetra docs repo].  You can also create an issue about a specific page by clicking the *Create documentation issue* button in the top right hand corner of the page.
+If you've found a problem in the docs, please create an issue in the https://github.com/tetrabiodistributed/project-tetra-docs/issues[Project Tetra docs repo]. You can also create an issue about a specific page by clicking the *Create documentation issue* button in the top right hand corner of the page.
```

Comment:
I'm personally a fan of double space after period, but whatever makes you sleep at night

Reply:
Tbh, I haven't really thought about it much :sweat_smile: